packages/eslint-plugin/src/rules/no-suppress-xcss/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { Rule } from 'eslint';
+
+export const noSuppressXCSS: Rule.RuleModule = {
+  meta: {
+    docs: {
+      recommended: true,
+      description:
+        'The xcss prop is predicated on adhering to the type contract. Supressing it breaks this contract and thus is not allowed.',
+      url: 'https://github.com/atlassian-labs/compiled/tree/master/packages/eslint-plugin/src/rules/no-supress-xcss',
+    },
+    messages: {
+      'no-suppress-xcss': 'bad!',
```

Comment:
```suggestion
      'no-suppress-xcss': '{{name}} should not contain a preceding {{tag}} as it breaks the type contract',
```

where name = the found `xcss` attribute and `tag` = `@ts-expect-error` or `@ts-ignore`

Reply:
Great suggestion, I would also suggest briefly mentioning the consequences if they ignore the lint error (e.g. "Ignoring this error will break the styling for this component.")